README.md

```diff
@@ -1,33 +1,183 @@
+[![Build status](https://circleci.com/gh/cahirwpz/demoscene.png)](https://circleci.com/gh/cahirwpz/demoscene)
+
+Setting up build environment
+---
+
+You need to reproduce the build environment I mentioned above. Fortunately
+_Dockerfiles_ list all commands required to do so, at least on _Debian 10
+(Buster)_ for _x86-64_ architecture. This is a good starting point for most of
+you.  Please only consider lines starting with _ADD_ and _RUN_ – please refer to
+[Dockerfile](https://docs.docker.com/engine/reference/builder/#run)
+documentation if needed.
+
+1. Start with [demoscene-toolchain](https://github.com/cahirwpz/demoscene-toolchain/blob/master/Dockerfile)
+   _Dockerfile_. When the toolchain is built you must set up your `PATH`
+   environment variable correctly. If you installed the toolchain in
+   `${HOME}/amiga` then add `export PATH=${PATH}:${HOME}/amiga/bin` to your
+   favorite shell initialization file (`.bashrc`, `.zshrc`, etc.). To validate
+   your setup, please issue `which m68k-amigaos-gcc` command which should print
+   where the compiler has been installed.
+
+2. Follow up with [demoscene](https://github.com/cahirwpz/demoscene/blob/master/Dockerfile)
+   _Dockerfile_. If you haven't had [Git LFS](https://git-lfs.github.com/)
+   installed previously, you'll have to issue `git lfs install` command. Then
+   clone *demoscene* repository once again to pull in binary files, otherwise
+   you'll get links instead of real data – which you can fix by issuing
+   `git lfs pull` command.
+
+The Amiga emulator
+---
+
+You need an emulator to test Amiga binaries. Luckily, [demoscene-toolchain](https://github.com/cahirwpz/demoscene-toolchain) 
+provides [fs-uae](https://fs-uae.net) emulator. Now, **important** thing –
+vanilla `fs-uae` doesn't provide features that are required for full experience.
+The emulator has been
+[patched](https://github.com/cahirwpz/demoscene-toolchain/tree/master/patches),
+which enables Amiga programs to efficiently output diagnostic messages using
+[UAE traps](https://github.com/cahirwpz/demoscene/blob/master/include/uae.h) and
+provides various fixes for debugger integration. The emulator is installed in
+the same directory as the compiler. Since this directory has already been added
+to your `PATH` environment variable you should be able to use it without
+troubles. To verify that, please issue `which fs-uae` command and confirm that
+it prints correct installation path.
+
+Compiling source code
+---
+
+Navigate to the [demoscene](https://github.com/cahirwpz/demoscene) repository
+you've just cloned and issue `make` command. If you build process fails, please
+verify that you have performed all steps listed above correctly. If `make`
+happens to complain about missing command – find the software package and
+install it. You cannot expect us to help you set up your environment!
+
+Running the effect under emulator
+---
+
+At first, set the correct path to a kickstart file in 
```

Comment:
Please tell a user she should figure out which of directories in her system stores kickstarts for `fs-uae`. When found she should copy kickstart files there and check their md5 checksums.

Reply:
Done via references to the fs-uae docs.